app/Models/Forum/Topic.php

```diff
@@ -199,33 +188,31 @@ public function moveTo($destinationForum)
             return false;
         }
 
-        return DB::transaction(function () use ($destinationForum) {
+        return $this->getConnection()->transaction(function () use ($destinationForum) {
             $originForum = $this->forum;
             $this->forum()->associate($destinationForum);
             $this->save();
 
-            $posts = $this->posts()->withTrashed()->with(['forum', 'topic'])->get();
-            $posts->each(function ($post) use ($destinationForum) {
-                $post->forum()->associate($destinationForum);
-                $post->save();
-            });
+            $this->posts()->withTrashed()->update(['forum_id' => $this->forum_id]);
```

Comment:
need to disable timestamps otherwise laravel tries to update an empty column with the time

Reply:
das coo